ethcore/src/block.rs

```diff
@@ -463,7 +471,7 @@ impl Drain for SealedBlock {
 }
 
 /// Enact the block given by block header, transactions and uncles
-fn enact(
+pub (crate) fn enact(
```

Comment:
```suggestion
pub(crate) fn enact(
```

(I think this is usual whitespace convention, but correct me if I'm wrong.)

Reply:
Yeah, I agree